client/my-sites/customer-home/cards/features/domain-upsell/test/index.jsx

```diff
@@ -67,9 +69,217 @@ jest.mock( '@automattic/domain-picker/src', () => {
 	};
 } );
 
+let pageLink = '';
+jest.mock( 'page', () => ( link ) => ( pageLink = link ) );
+jest.mock( 'calypso/state/checklist/actions' );
+
 const domainUpsellHeadingFreePlan = 'Own your online identity with a custom domain';
+const domainUpsellHeadingPaidPlan = 'Make your mark online with a memorable domain name';
+const buyThisDomainCta = 'Buy this domain';
+const searchForDomainCta = 'Search for another domain';
 
 describe( 'index', () => {
+	test( 'Should show H3 content for the Home domain upsell and test search domain button link', async () => {
```

Comment:
Thanks for also update the tests, amazing!

Reply:
I just recover re-use your tests 😉